app/uploaders/image_uploader.rb

```diff
@@ -6,25 +6,30 @@ class ImageUploader < WhitehallUploader
   end
 
   def extension_white_list
-    %w(jpg jpeg gif png)
+    %w(jpg jpeg gif png svg)
   end
 
-  version :s960 do
+  version :s960, if: :image? do
     process resize_to_fill: [960, 640]
   end
-  version :s712, from_version: :s960 do
+  version :s712, from_version: :s960, if: :image? do
     process resize_to_fill: [712, 480]
   end
-  version :s630, from_version: :s712 do
+  version :s630, from_version: :s712, if: :image? do
     process resize_to_fill: [630, 420]
   end
-  version :s465, from_version: :s630 do
+  version :s465, from_version: :s630, if: :image? do
     process resize_to_fill: [465, 310]
   end
-  version :s300, from_version: :s465 do
+  version :s300, from_version: :s465, if: :image? do
     process resize_to_fill: [300, 195]
   end
-  version :s216, from_version: :s300 do
+  version :s216, from_version: :s300, if: :image? do
     process resize_to_fill: [216, 140]
   end
+
+  def image?(new_file)
```

Comment:
Wondering about naming here... could be a bit confusing if we're saying that an SVG is not an image (and it that case it shouldn't be in the ImageUploader)? Could we instead ask something like `has_thumbnail?` and base it on a thumbnail whitelist (I guess basically the white list but excluding 'svg')? Would that get around the necessity of passing the second argument to the test helpers in the other test files? Haven't done anything with Carrierwave before though so not sure if this helpful...